### PR TITLE
LIVE-1004 Hedera memo + LLM integration

### PR DESCRIPTION
**Pull request by haammar-ledger**
### 📝 Description

Hedera send/receive integration on LLM
Memo support on both LLD and LLM

### ❓ Context

- **Impacted projects**: LLC + LLD + LLM
  
- **Linked resource(s)**:
  - https://ledgerhq.atlassian.net/browse/LIVE-1004
  - https://ledgerhq.atlassian.net/browse/LIVE-1096
  - https://ledgerhq.atlassian.net/browse/LIVE-2701
  - https://ledgerhq.atlassian.net/browse/LIVE-2892
  - https://ledgerhq.atlassian.net/browse/LIVE-2410

### ✅ Checklist

- [x] **Test coverage**: _Did you write any tests to cover the changes introduced by this pull request?_
- [x] **Atomic delivery**: _Is this pull request standalone? In order words, does it depend on nothing else?_
- [x] **No breaking changes**: _Does this pull request contain breaking changes of any kind? If so, please explain why._

### CODE REVIEW

File: apps/ledger-live-mobile/scripts/sync-families-dispatch.sh [hunk 1]
```diff
@@ -4,18 +4,21 @@ set -e
...
+AccountSubHeader.js \
+SendAmountFields.js \
+screens.js \
+SendRowsCustom.js \
+SendRowsFee.js \
+AccountBalanceSummaryFooter.js \
+SubAccountList.js \
+Confirmation.js \
+ConnectDevice.js \
+NoAssociatedAccounts.js
```
Comment (haammar-ledger) [inline apps/ledger-live-mobile/scripts/sync-families-dispatch.sh:21]: @jackthta Can you please remind me what that change was for? Is it necessary?

Comment (jackthta) [reply apps/ledger-live-mobile/scripts/sync-families-dispatch.sh:21]: @haammar-ledger 

Oh yes, this change is for the custom components defined in `apps/ledger-live-mobile/src/families/hedera/*`. They are to be injected via `byFamily` in their respective screens:
- `apps/ledger-live-mobile/src/screens/AddAccounts/03-Accounts.js`
- `apps/ledger-live-mobile/src/screens/ReceiveFunds/02-ConnectDevice.js`
- `apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.js`

**Comment by haammar-ledger** [comment]
@jackthta Can you please add mutations to the bot's spec file, to test transactions that include a memo?

**Comment by aelmilligy-ledger** [comment]
@haammar-ledger I approved the review. The codecov report shows a decrease on the coverage, could I have your feedback on it ?

**Review by henrily-ledger — APPROVED** [review]

**Pull request merged**